packages/react-components/react-breadcrumb/src/utils/truncateLongContent.test.ts

```diff
@@ -0,0 +1,40 @@
+import { truncateLongName, truncateLongTooltip } from './truncateLongContent';
+
+const defaultTestNames = [
+  ['', ''],
+  ['Regular name', 'Regular name'],
+  ['Name which is longer than 30 characters', 'Name which is longer than 30 c...'],
+  ['Name which equal 30 characters', 'Name which equal 30 characters'],
+];
+
+describe('truncate method', () => {
+  it.each(defaultTestNames)('truncates long name correctly', (name, expected) => {
```

Comment:
while the `each` makes things more "polished" it drastically downgrades tests readability. One has no idea what is being tested when checking logs.

Also this combo `['', ''],`  might feel unintuitive to reader. 

<img width="1034" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/3a76b266-b6c1-4a6d-a2c6-7d32e1a2e18f">


Can we improve this ?

Reply:
That's strange, because when I run it locally it shows the case (when it fails :)). I'll try to improve it. If not, I'll make a regular test

Reply:
I used regular tests. Also, it's more consistent now